Classes/common/touchdb/TDRemoteRequest.m

```diff
@@ -174,80 +183,22 @@ - (BOOL)retry
     // request might have succeeded on the remote server, and by retrying we'd be issuing it again.
     // PUT and POST requests aren't generally idempotent, but the ones sent by the replicator are.
 
-    if (_retryCount >= kMaxRetries) return NO;
-    NSTimeInterval delay = RetryDelay(_retryCount);
-    ++_retryCount;
-    CDTLogVerbose(CDTTD_REMOTE_REQUEST_CONTEXT, @"%@: Will retry in %g sec", self, delay);
-    [self startAfterDelay:delay];
-    return YES;
+//    if (_retryCount >= kMaxRetries) return NO;
```

Comment:
Retires should be handled by `CDTURLSession` I haven't figured out how though. The authentication methods changed, most of it is still present, but for accepting SSL certs currently only the default NSURLSession behaviour is observed. We should have it back to feature parity with the previous implementation but I wonder if there was a way to configure the SSL cert verification anyway